docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,34 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the requirements, it's possible to hotplug an interface by migrating the VM.
+The actual attachment wont take place immediately, the new interface will be ready once the migration is completed.
+
+#### Add new interface:
+```bash
+virtctl addinterface vmi-fedora --network-attachment-definition-name new-fancy-net --name dyniface1
+```
+At this point the new interface is added to the spec but isn't attached to the VM. 
+
+#### Migrate the VM:
+```bash
+cat <<EOF kubectl apply -f -
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstanceMigration
+metadata:
+  name: migrate-vmi-fedora
+spec:
+  vmiName: vmi-fedora
+EOF
+```
+Or using `virtctl`:
+```bash
+virtctl migrate vmi-fedora
+```
```

Comment:
nit: Please consider using this method first.
I personally find it simpler.

Reply:
I removed the manifest method and added a link to the full live migration doc.